<launch>
    <node pkg="manipulation_planning" type="manipulation_planning_cspace_test" name="manipulation_planning_cspace_test_node" args="panda0_arm" output="screen"/>
</launch>